news/2022-07-06-new-featured-artists-rootkit-grant.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Rootkit & Grant"
+date: 2022-07-06 10:00:00 +0000
+---
+
+Two flagship members of the [**Monstercat**](https://osu.ppy.sh/beatmaps/artists/255) repertoire sail into our Featured Artist collection!
+
+![](/wiki/shared/news/2022-07-06-new-featured-artists-rootkit-grant/rootkit-grant-banner.jpg)
+
+**Rootkit** and **Grant** are both masters of their crafts. While **Rootkit** has a drum & bass throughline connecting all of his diverse tunes, **Grant** explores a number of electronic designs. Of course, both of their styles have already proven themselves to be perfect fits for the osu! community, through beatmaps like [Rootkit - Against The Sun (feat. Anna Yvette)](https://osu.ppy.sh/beatmapsets/340263) or [Grant - Starship](https://osu.ppy.sh/beatmapsets/609900)!
+
+We're working with *a lot* of artists through **Monstercat**, some of which have bigger tracklists than others. Because both of these lists are on the slimmer side, we've decided to pair them together in this announcement!
+
+That said, check out [**Rootkit**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/280) and [**Grant**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/281) for **6** new song additions!
+
+Too busy to play through some maps on your own? See how these songs translate to mapping through this quick video, then check out the other previews below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/280/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Rootkit - Against The Sun (feat. Anna Yvette)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Grant - Starship
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Rootkit - Do It
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### Grant - Weapon (feat. Baum)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+As soon as you're ready to start mapping (also known as *right now*), sort through [**Rootkit**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/280) and [**Grant**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/281)! This two-for-one deal won't come by often!
+
+In case you missed our [Tracks Updates news post](), we've also launched a new round of [**Featured Artist playlists**](/wiki/Featured_Artists/Featured_Artist_playlists)! Check those out in osu!(lazer) if you're at all curious about the long-term future of osu! (and maybe check out the [@osugame Twitter](https://twitter.com/osugame) if you're curious about the short term).
```

Comment:
```suggestion
In case you missed our [Tracks Updates news post](https://osu.ppy.sh/home/news/2022-07-02-featured-artist-track-updates-summer-2022), we've also launched a new round of [**Featured Artist playlists**](/wiki/Featured_Artists/Featured_Artist_playlists)! Check those out in osu!(lazer) if you're at all curious about the long-term future of osu! (and maybe check out the [@osugame Twitter](https://twitter.com/osugame) if you're curious about the short term).
```